bootstrap/kubeadm/controllers/kubeadmconfig_controller.go

```diff
@@ -436,9 +437,28 @@ func (r *KubeadmConfigReconciler) handleClusterNotInitialized(ctx context.Contex
 		InitConfiguration:    initdata,
 		ClusterConfiguration: clusterdata,
 		Certificates:         certificates,
-	})
+	}
+
+	var cloudInitData []byte
```

Comment:
We should rename this variable to something more proper, it doesn't necessarily contain cloud-init data

Reply:
Done, will push soon.